Test_Feedback_form/script.js

```diff
@@ -0,0 +1,41 @@
+function showUsersTable(fields = []) {
+
+    let numberOfTheRecord = 0;
+
+    let innerHtmlTableBody = '';
+    for (let i = 0; i < fields.length; i++) {
+        innerHtmlTableBody += '<tr><td>' + fields[i].name + '</td><td>' + fields[i].email + '</td><td>' + fields[i].dateOfAddedFeedback + '</td></tr>';
+        numberOfTheRecord++
```

Comment:
@Ovsiannikov78 You don't need to use additional records counter - you already have this number in `fields.length`. BTW, fields is not the best name for this variable: field relates to the column, but we mean number of rows.